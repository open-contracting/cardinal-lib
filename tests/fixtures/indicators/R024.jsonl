{"ocid":"F","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"1","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"2","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"3","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"4","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"5","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"6","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"accept","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":101,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"cancelled","suppliers":[{"id":"C"}]},{"status":"unsuccessful","suppliers":[{"id":"U"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"pending award","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"pending"}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"invalid award","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"invalid"}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"invited bid","bids":{"details":[{"status":"invited","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"pending bid","bids":{"details":[{"status":"pending","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"withdrawn bid","bids":{"details":[{"status":"withdrawn","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"disqualified bid","bids":{"details":[{"status":"disqualified","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"non-price criteria","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":2,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"multiple awards","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]},{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"multiple tenderers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"},{"id":"M"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"multiple suppliers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"},{"id":"X"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"cancelled tender","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024","status":"cancelled"}}
{"ocid":"repeat tenderers","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":2,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"R024"}}
{"ocid":"no price comparison","bids":{"details":[{"status":"valid","tenderers":[{"id":"W"}],"value":{"amount":1,"currency":"USD"}},{"status":"valid","tenderers":[{"id":"L"}],"value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active","suppliers":[{"id":"W"}]}],"tender":{"procurementMethodDetails":"NPC"}}
