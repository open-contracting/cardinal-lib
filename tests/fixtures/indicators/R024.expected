{"OCID":{"0":{"R024":0.0}}}
